pkg/authorization/tokenreviewauthorizer/tokenreviewauthorizer.go

```diff
@@ -0,0 +1,29 @@
+package tokenreviewauthorizer
+
+import (
+	"context"
+
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+)
+
+type tokenReviewAuthorizer struct{}
+
+func (tokenReviewAuthorizer) Authorize(ctx context.Context, a authorizer.Attributes) (authorized authorizer.Decision, reason string, err error) {
+	if a.GetUser().GetName() != "system:serviceaccount:openshift-authentication:tokenreviewer"{
```

Comment:
alright, so this is a service account token used by the KAS, right?

Reply:
> alright, so this is a service account token used by the KAS, right?

In @stlaz's PR, yes.

Reply:
Got it. This is just an example. Thanks for sharing it with me.

lgtm